Gathering handlers from Windows systems

execgit opened the pull request:
Powershell scripts to gather handles, as well as example json

execgit committed:
Initial PowerShell script for listing handlers in Windows

Tested with Windows 7, PowerShell 2

execgit committed:
Change app dict to list, modified example accordingly

execgit committed:
Fixed bugs: output on Win10, command line handling

execgit committed:
Modified to work with Server2008

Had trouble locating System.ServiceModel.Web on some systems.
Worked around this by reimplementing the JSON serialization of
registry values. The new implementation requires a ToString
method from the values, which has not been a problem in testing.

execgit committed:
Stripped directory from handler path. Add utf-16 support to merge.

execgit committed:
Modified example with new handlers-list.ps1

execgit committed:
Flake8 fix

oherrala commented on windows/handlers-list.ps1:
```diff
@@ -0,0 +1,121 @@
+	[Parameter(mandatory=$true)][string]$output
+)
+
+# Invoke with:
```
This useful information should be moved to `README.md`.

execgit committed:
Updated handlers-example.json with new launchers.

execgit committed:
Added README.md for windows

Thanks to @oherrala for the suggestion.

execgit committed:
Fixed the handling of encodings in handlers-merge.py

It failed in some edge cases.

execgit committed:
Added handlers

execgit committed:
Script tested with Windows 8.1

execgit committed:
Added handlers

oherrala commented on windows/handlers-example.json:
```diff
@@ -53,13 +62,28 @@
                 "registrykey": "HKEY_CLASSES_ROOT\\DLNA-PLAYSINGLE"
             },
             {
```
Is this entry duplicate with next entry below? There's also other duplicate entries it seems.

execgit committed:
handlers-merge.py: Added option to force lowercase

Data related to handlers is case-insensitive in the Windows registry.
Added an option to handlers_merge to lowercase the input before
merging to avoid duplicate handlers.

execgit committed:
Added handlers, removed duplicates

Used the option -l in handlers-merge.py to remove duplicates

execgit committed:
Moved JSON lowercasing to handlers-list.ps1

Reverted commit 9a5fdec7767547b598b705842c86fbe8d2efc248. Added
lowercasing to handlers-list.ps1 instead.

execgit committed:
handlers-list.ps1: Add colon

URL handlers generated by handlers-list should now include a colon,
i.e. http: instead of http

execgit committed:
Modified handlers-example.json according to fd6cdc85f24bc325c5d5942f87d9d65baae1415c

execgit committed:
Added handlers from a clean Asus install.

execgit committed:
Added handlers from a default HP install.

execgit committed:
Added handlers
